automatic/opera/opera.nuspec

```
@@ -30,6 +30,10 @@ For example: `--params '"/NoAutostart /NoDesktopShortcut /NoTaskbarShortcut"'`
     <releaseNotes>https://blogs.opera.com/desktop/changelog-for-102/#b4880.40</releaseNotes>
     <tags>browser opera cross-platform internet admin</tags>
     <packageSourceUrl>https://github.com/chocolatey-community/chocolatey-packages/tree/master/automatic/opera</packageSourceUrl>
+    <dependencies>
+      <dependency id="chocolatey-core.extension" version="1.3.4" />
```

As this is a new dependency, set it to the latest version available.

```suggestion
      <dependency id="chocolatey-core.extension" version="1.4.0" />
```

---

Addressed in https://github.com/chocolatey-community/chocolatey-packages/pull/2308/commits/1526a54c0318c9066e2483d15cf90fd9e57c2898.